.baseline/checkstyle/checkstyle.xml

```diff
@@ -92,6 +92,7 @@
                 org.apache.iceberg.IsolationLevel.*,
                 org.apache.iceberg.NullOrder.*,
                 org.apache.iceberg.MetadataTableType.*,
+                org.apache.iceberg.spark.SparkTableUtil.*,
```

Comment:
Is it worth changing this just to reuse `MapJoiner`? I'd say it is easier to simply define one more `MapJoiner` as we do in a couple of other places.

Reply:
So far Majoiner is only defined in one other place, I was actually wondering whether I should move it to a global util class. I think duplicating here is fine

Reply:
I think duplicating it for now is OK. That way, we don't have to maintain one more utility class.